spec/integration/recipe_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts index page', type: :system do
+  before do
+    driven_by(:rack_test)
+    @user = User.create(name: 'admin',
+                        email: 'admin@email.com',
+                        password: 'password', password_confirmation: 'password')
+    @user.save
+    @recipe = Recipe.create(user_id: @user.id, name: 'Recipe 1', preparation_time: 10, cooking_time: 20,
+                            description: 'This is test recipe and it is very delicious', public: true)
+    visit 'user/sign_in'
+    within('#new_user') do
+      fill_in 'Email', with: 'admin@email.com'
+      fill_in 'Password', with: 'password'
+    end
+    find_button('Log in').click
+  end
+
+  it 'I can browse the public recipes.' do
+    expect(page).to have_content('Signed in successfully.')
+    expect(page).to have_content('Recipe 1')
+  end
```

Comment:
- [ ] This test is failing. Please take a look at the error below and try to fix it 👇
![image](https://user-images.githubusercontent.com/46839548/189492327-cafbc7b7-4a23-455a-b436-619eb381bbfb.png)

Reply:
Solved